soccer/gameplay/skills/capture.py

```diff
@@ -1,270 +0,0 @@
-            'dist to ball < threshold')
-
-        self.add_transition(
-            Capture.State.fine_approach,
-            Capture.State.delay,
-            lambda: self.robot.has_ball, #evaluation.ball.robot_has_ball(self.robot),
-            'Has ball')
-
-        self.add_transition(
-            Capture.State.delay,
-            behavior.Behavior.State.completed,
-            lambda: time.time() - self.start_time > Capture.DelayTime and
-                    self.robot.has_ball, #evaluation.ball.robot_has_ball(self.robot),
-            'Delay before finish')
-
-        self.add_transition(
-            Capture.State.delay,
-            Capture.State.fine_approach,
-            lambda: not self.robot.has_ball, #not evaluation.ball.robot_has_ball(self.robot),
-            'Lost ball during delay')
-
-        self.add_transition(
-            Capture.State.fine_approach, Capture.State.coarse_approach,
-            lambda: not (self.bot_in_front_of_ball() or
-                        self.bot_near_ball(Capture.CoarseToFineApproachDistance)) and
-                    (not self.bot_near_ball(Capture.CoarseToFineApproachDistance * 1.5) or
-                    not main.ball().pos),
-            'Lost ball during delay')
-
-        self.dribbler_power = Capture.DribbleSpeed
-
-        self.lastApproachTarget = None
-        self.faceBall = faceBall
-
-    def bot_to_ball(self):
-        return main.ball().pos - self.robot.pos
-
-    def bot_near_ball(self, distance):
-        return (self.bot_to_ball().mag() < distance)
-
-    # Ball is moving towards us and will not stop before reaching us
-    def bot_in_front_of_ball(self):
-        ball2bot = self.bot_to_ball() * -1
-        return (ball2bot.normalized().dot(main.ball().vel) > Capture.InFrontOfBallCosOfAngleThreshold) and \
-               ((ball2bot).mag() < (evaluation.ball.predict_stop() - main.ball().pos).mag())
-
-    # calculates intercept point for the fast moving intercept state
-    def find_moving_intercept(self):
-        return find_robot_intercept_point(self.robot)
-
-    # returns intercept point for the slow moving capture states
-    def find_intercept_point(self):
-        return find_robot_capture_point(self.robot)
-
-    def execute_running(self):
-        self.robot.set_planning_priority(planning_priority.CAPTURE)
-
-        if (self.faceBall):
-            self.robot.face(main.ball().pos)
-
-    # sets move subbehavior
-    def execute_intercept(self):
-        self.robot.set_dribble_speed(self.dribbler_power)
-        pos = self.find_moving_intercept()
-        self.robot.move_to(pos)
-
-    def on_enter_coarse_approach(self):
-        self.lastApproachTarget == None
-
-    def execute_coarse_approach(self):
-        pos = self.find_intercept_point()
-
-        # Don't move our target if it's somewhat close to our old target
-        if (self.lastApproachTarget != None and
-            (pos - self.lastApproachTarget).mag() < 0.1):
```

Comment:
Next time, please refactor this magic number to a constant before deleting it please